Description: remove no longer valid ifdef in test

Author: Wang Zichong <wangzichong@deepin.org>

---
Origin: other
Forwarded: not-needed
Reviewed-By: Wang Zichong <wangzichong@deepin.org>
Last-Update: 2023-05-16

--- kcodecs-5.90.0.orig/autotests/kusasciitextcodectest.cpp
+++ kcodecs-5.90.0/autotests/kusasciitextcodectest.cpp
@@ -46,13 +46,8 @@ void KUsAsciiTextCodecTest::testBrokenBu
     QTextCodec::ConverterState failConverterState;
 
     const QByteArray failEncoded8Bit = qtCodec->fromUnicode(failUnicodeText.constData(), failUnicodeText.length(), &failConverterState);
-    // Bug seems fixed in Qt6
-#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+
     QCOMPARE(failConverterState.invalidChars, 1);
-#else
-    // confirm the broken result 0, which rather should be 1
-    QCOMPARE(failConverterState.invalidChars, 0);
-#endif
 }
 
 void KUsAsciiTextCodecTest::testEncoding()
